docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -454,10 +602,104 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example of the aforementioned command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your signer is registered
+
+After installing the Mithril Signer, you can verify that your node is registered by checking your Mithril Signer node logs.  
+
+First, download the script in the desired directory:
```

Comment:
Should it be 'into' the desired directory? Currently, it feels like we need to say 'from', but that does not seem to be right. If you choose to update this, please consider other similar instructions below.

Reply:
Thanks, 'into' fits better indeed.